programs/mango-v4/src/instructions/perp_place_order.rs

```diff
@@ -109,8 +110,32 @@ pub fn perp_place_order(ctx: Context<PerpPlaceOrder>, order: Order, limit: u8) -
 
     let now_ts: u64 = Clock::get()?.unix_timestamp.try_into().unwrap();
 
-    // TODO apply reduce_only flag to compute final base_lots, also process event queue
-    require!(order.reduce_only == false, MangoError::SomeError);
+    let effective_pos = account
+        .perp_position(perp_market_index)?
+        .effective_base_position_lots();
+    if perp_market.is_reduce_only() {
+        require!(
```

Comment:
hmm I saw something similar in v3, basically for spot orders, we restricted user to be able to have atmost 1 open order after market went into reduce only, wdyt?